src/lesson4/task1/Main.java

```diff
@@ -0,0 +1,31 @@
+package lesson4.task1;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        int[] array = new int[5];
+        Random r = new Random();
+        for (int i = 0; i < array.length; i++) {
+            array[i] = r.nextInt(100);
+            //System.out.print(array[i] + " ");
+        }
+        System.out.println("Введите число: ");
+        Scanner sc = new Scanner(System.in);
+        int g = sc.nextInt();
+        //for (int i : array) {
+        int a = 0;
+        for (int j : array) {
+            if (j == g) {
+                System.out.println("Введённое число есть в массиве");
+                a++;
```

Comment:
It's better to use boolean instead of int as flag.
Also do we need to continue loop, when we find the first occurrence of the number?